src/main/java/com/couchbase/lite/router/Router.java

```diff
@@ -1264,6 +1262,7 @@ public Status do_POST_Document_revs_diff(Database _db, String _docID, String _at
 
         // Look them up, removing the existing ones from revs:
         try {
+            // query db only, not necessary to be syncrhonized
```

Comment:
This comment might not be correct if want to make calling to the database object thread-safe.